app/pages/login/components/authForm.tsx

```diff
@@ -9,6 +9,7 @@ interface AuthFormProps {
   isSignUp: boolean;
   error?: string;
   onSubmit: (data: {
+      username: string;
     email: string;
     password: string;
     type: SubmitAction;
```

Comment:
ok. 
<img width="1299" alt="Screenshot 2024-12-15 at 00 02 02" src="https://github.com/user-attachments/assets/020830a6-1420-4913-9e95-14ab901601d0" />
Install prettier

<img width="1299" alt="Screenshot 2024-12-14 at 23 57 47" src="https://github.com/user-attachments/assets/dc344e29-14ce-4c9f-a60e-a44a5e727b86" />


add checkbox here *Run on save*

you can also click to Actions on save 
 
<img width="1299" alt="Screenshot 2024-12-15 at 00 02 37" src="https://github.com/user-attachments/assets/365b4432-2e06-4524-a7fe-b8ccc10c5bc4" />

and check everything

********************
In js/ts world Prettier is a code formatter (super popular one). it goes through files and fixes styles.

Reply:
by default intelij has its own ` prettier `, so your code is kinda styled. the problem is when you are in a team and everybody has different styling settings. You intelij settings cant be shared with Vscode gays , sry guys  , so it is better to have a single config for all in one place. thats why prettier exists. Also prettier has cli tools so you can apply pretttier to all files with 1 line of code in console